OracleEDQ/README.md

```diff
@@ -0,0 +1,120 @@
+
+If a proxy is needed for the host to access yum.oracle.com during build, then first set up the appropriate environment, e.g.:
+
+        $ export http_proxy=myproxy.example.com:80
+        $ export https_proxy=myproxy.example.com:80
+        $ export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com"
+
+Build the image:
+
+        $ cd OEDQ/dockerfiles
+        $ ./buildDockerImage.sh -v 12.2.1.3.0
+
+## Creating an Oracle Enterprise Data Quality Container
+
+The OEDQ image provides a script to create a OEDQ domain, with the Director application.
+
+This script is invoked when a container is started using the image. If the container is subsequently restarted, then the OEDQ domain is restarted automatically.
+
+### Starting a new EDQ container
+The following variables are mandatory when starting the container:
+* ADMIN_USERNAME - WebLogic admin username for the new domain
+* ADMIN_PASSWORD - WebLogic admin password
+* DB_HOST - Host name for database into which new schemas will be created
+* DB_PORT - Database listener port
+* DB_SERVICE - Database instance service name
+* DB_USERNAME - Database sysdba username
+* DB_PASSWORD - Database sysdba password
+* SCHEMA_PREFIX - Schema prefix for new schemas for the new domain. This must not already be in use, otherwise domain creation will fail.
+* SCHEMA_PASSWORD - Password for all new schemas created
+
+ORACLE_HOME is fixed to `/u01/oracle`. 
+DOMAIN_HOME is fixed to ` $HOST_VOLUME: /u01/oracle/user_projects/domains/base_domain`
+Where HOST_VOLUME  stands for a directory on the host where you map your domain directory and both the Admin Server and Managed Server containers can read/write to.
+DB_HOST - Host name for database into which new schemas will be created
+DB_PORT - Database listener port 
+DB_SERVICE - Database instance service name
+RCUPREFIX - Schema prefix for new schemas for the new domain. This must not already be in use, otherwise domain creation will fail.
+DB_PASSWORD - Database sysdba password
+
+DB_SCHEMA_PASSWORD- Password for all new schemas created
+DOMAIN_PASSWORD= WebLogic admin username for the new domain
+
+Note: Grant read, write and execute permission to all for /scratch/xyz directory
+
+e.g.:
+
+        $ docker run --name EDQAS -it -p 7001:7001 --env-file ./12.2.1.3.0/edq.db_env.list -v /scratch/xyz:/u01/oracle/user_projects oracle/edq:12.2.1.3.0
+
+On success, the following is logged: `The configuration of Oracle Fusion Middleware completed successfully.`. The script then tails the domain log to stdout.
+
+
+### Starting another container to Start EDQ Managed Server
+
+Start a container to launch the Managed Server from the image created in prior step  The environment variables used to run the Managed Server image are defined in the file EDQServer.env.list.
```

Comment:
Why are you requiring a file when you just have two variables? Why not just define them in the `Dockerfile`?

Reply:
File EDQServer.env.list removed. As it is replaced by command line paramerters.